pandas/core/strings.py

```diff
@@ -1095,24 +1095,61 @@ def str_pad(arr, width, side='left', fillchar=' '):
 
 def str_split(arr, pat=None, n=None):
     """
-    Split each string (a la re.split) in the Series/Index by given
-    pattern, propagating NA values. Equivalent to :meth:`str.split`.
+    Split strings around given separator/delimiter.
+
+    Split each string in the Series/Index by given
+    pattern, propagating NaN values. Equivalent to :meth:`str.split`.
 
     Parameters
     ----------
     pat : string, default None
-        String or regular expression to split on. If None, splits on whitespace
+        String or regular expression to split on.\
```

Comment:
Hmm does this render the backslash in the output?

Reply:
Oh OK good to know. That said, I don't think the line real is required or part of the standard (see the **kwargs example in first instance of `class Series` in the sprint documentation <https://python-sprints.github.io/pandas/guide/pandas_docstring.html>)

Should be fine just to place on separate line and ensure proper indentation